src/devices/machine/ncr5385.cpp

```diff
@@ -245,6 +246,9 @@ u8 ncr5385_device::aux_status_r()
 
 	if (!m_int_status)
 	{
+		// AB mask out any bits
+		data &= ~(AUX_STATUS_MSG | AUX_STATUS_CD | AUX_STATUS_IO);
```

Comment:
I don't think this should be necessary; resetting and latching of these bits should be properly controlled by changes to the interrupt status. Did you find a bug in the logic?